tfx/components/example_gen/big_query_example_gen/executor.py

```diff
@@ -51,13 +51,16 @@ def RowToExample(self, instance: Dict[Text, Any]) -> tf.train.Example:
         feature[key] = tf.train.Feature()
       elif data_type in ('INTEGER', 'BOOLEAN'):
         feature[key] = tf.train.Feature(
-            int64_list=tf.train.Int64List(value=[value]))
+            int64_list=tf.train.Int64List(value=value if isinstance(value, list) else [value]))
```

Comment:
can we check the REPEATED instead of isinstance(value, list)?

Reply:
Not without redesigning it.

We get the schema in the __init__ and then we only have the row here, which doesn't say anything about the [original BQ type](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.table.Row.html).